src/Symfony/Component/HttpFoundation/FileBag.php

```diff
@@ -65,32 +65,28 @@ public function add(array $files = [])
     /**
      * Converts uploaded files to UploadedFile instances.
      *
-     * @param array|UploadedFile $file A (multi-dimensional) array of uploaded file information
-     *
      * @return UploadedFile[]|UploadedFile|null A (multi-dimensional) array of UploadedFile instances
      */
-    protected function convertFileInformation($file)
+    protected function convertFileInformation(mixed $file)
```

Comment:
Shall we backport the changes you've made to this method?

Reply:
we might backport in a follow up